### ISSUE TITLE
How to get rid of banner notice in admin area?

### ISSUE THREAD

**Issue by davidhealey** [usefulteam/jwt-auth#136]
This one

![image](https://github.com/user-attachments/assets/5b4de358-bf2d-4277-aa6a-451d5a093030)

I'm on the latest version of the plugin, and I have clicked the X, and still the banner shows up.

**Comment by dominic-ks** [comment]
@davidhealey Hello, that's odd since the banner shouldn't show if you're on the latest, but can you check in your developer console to see if the ajax request is successful when you close it?

**Comment by davidhealey** [comment]
I see nothing in the console when I click the X.

**Comment by davidhealey** [comment]
Seems to be related to [this](https://github.com/usefulteam/jwt-auth/blob/master/class-update.php#L64) failing to work. Any suggestions?

**Comment by dominic-ks** [comment]
Hello, to clarify what I was asking for before, can you see that when you click the x an XHR request is sent to your server on the network tab?

**Comment by davidhealey** [comment]
> Hello, to clarify what I was asking for before, can you see that when you click the x an XHR request is sent to your server on the network tab?

Yes I see the xhr request. Is there any particular info you want from it?
